packages/amplication-server/src/core/entity/entity.service.ts

```diff
@@ -629,9 +630,8 @@ export class EntityService {
   }
 
   async updateFieldDataTypeIdByRelatedEntity(
```

Comment:
let's refactor the method name to reflect what it does. Something like `getRelatedFieldScalarTypeByRelatedEntityIdType`